slack_bolt/context/base_context.py

```diff
@@ -103,13 +108,28 @@ def matches(self) -> Optional[Tuple]:
         """Returns all the matched parts in message listener's regexp"""
         return self.get("matches")
 
+    @property
+    def function_execution_id(self) -> Optional[str]:
+        """The `function_execution_id` associated with this request. Only available for function related events"""
+        return self.get("function_execution_id")
+
+    @property
+    def inputs(self) -> Optional[Dict[str, Any]]:
+        """The `inputs` associated with this request. Only available for function related events"""
```

Comment:
```suggestion
        """The bot token resolved for this function request. Only available for `function_executed` and interactivity events scoped to a custom step."""
```

Reply:
I don't think this is the `bot token` I think this is the inputs to the function 🤔

Reply:
o wtf. this might have been a copy-paste error. my mistake!

Reply:
Edited the suggestion above!